### PR TITLE
doc: note about bounty

### PR DESCRIPTION
**Pull request by pbludov**
Some information for volunteers. I don't really know what's supposed to be there. Suggestions and comments are welcome.

### CODE REVIEW

File: src/xdocs/contributing.xml [hunk 1]
```diff
@@ -169,5 +169,20 @@
         feel free to send a reminder email about your tracker item.
       </p>
     </section>
+
+    <section name="Bounty">
+
+      <p>
+        Some (mostly important) issues are marked with the <b>bounty</b> label.
```
Comment (romani) [inline src/xdocs/contributing.xml]: Please add link to gihub filter by bounty label.
A lot of user still unfamiliar with GitHub features.

**Comment by pbludov** [comment]
image

File: src/xdocs/contributing.xml (hunk 1 above)
Comment (pbludov) [reply src/xdocs/contributing.xml]: done

**Comment by rnveach** [comment]
@romani please approve.

**Pull request merged**